proto/process/agent.proto

```diff
@@ -24,6 +24,7 @@ message ResCollector {
 
 message CollectorProc {
 	string  hostName = 2;
+	string networkId = 11;
```

Comment:
Do we want to try to align the field IDs for `CollectorProc` and `CollectorContainer` so that we can _potentially_ move to a "skinnier" message type if needed in the future?